homeassistant/components/media_player/services.yaml

```diff
@@ -171,6 +171,16 @@ shuffle_set:
       description: True/false for enabling/disabling shuffle.
       example: true
 
+repeat_set:
+  description: Set repeating state.
+  fields:
+    entity_id:
+      description: Name(s) of entities to set.
+      example: 'media_player.spotify'
+    repeat:
+      description: True/false for enabling/disabling repeat.
```

Comment:
Is true/false the correct value ? What about "repeat 1" ?

Reply:
And yes, that also means we need to list repeat modes that the media player supports.

Reply:
Hmm maybe we should just not support "repeat 1" for now

Reply:
You're right, i will go back to development. Thanks